apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2VectorTestIT.java

```diff
@@ -0,0 +1,132 @@
+package io.stargate.sgv2.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.stargate.sgv2.common.IntegrationTestUtils;
+import io.stargate.sgv2.common.testresource.StargateTestResource;
+import io.stargate.sgv2.restapi.service.models.Sgv2ColumnDefinition;
+import io.stargate.sgv2.restapi.service.models.Sgv2Table;
+import io.stargate.sgv2.restapi.service.models.Sgv2TableAddRequest;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@QuarkusIntegrationTest
+@QuarkusTestResource(StargateTestResource.class)
+public class RestApiV2VectorTestIT extends RestApiV2QIntegrationTestBase {
+  public RestApiV2VectorTestIT() {
+    super("vec_ks_", "vec_t_", KeyspaceCreation.PER_CLASS);
+  }
+
+  @BeforeAll
+  public static void validateRunningOnVSearchEnabled() {
+    assumeThat(IntegrationTestUtils.supportsVSearch())
+        .as("Test disabled if backend does not support Vector Search (vsearch)")
+        .isTrue();
+  }
+
+  /*
+  /////////////////////////////////////////////////////////////////////////
+  // Tests: Create with Vector, index
+  /////////////////////////////////////////////////////////////////////////
+   */
+
+  @Test
+  public void tableCreateWithVectorIndex() {
+    final String tableName = testTableName();
+    createVectorTable(testKeyspaceName(), tableName);
+  }
+
+  /*
+  /////////////////////////////////////////////////////////////////////////
+  // Tests: Create, fail
+  /////////////////////////////////////////////////////////////////////////
+   */
+
+  // TODO: Add test trying to create vector column with non-float element
```

Comment:
should we try to complete these now?

Reply:
These are markers for #2715 so I am thinking of that as follow-up.